Give user feedback on progess of the task.

Moremackles opened the pull request:
It would be nice if the user had more feedback as to the progress of the task. Since you already have most of these variables it's really as simple as adding some sort of feedback inside of the loop or inside of the downloadChunk function. Note that this is my first time doing this and if I'm doing anything wrong please let me know.

ArneVogel commented:
Hey, you can test your changes after you installed [go](https://golang.org/dl/) with `go build`. The `go` keyword in go starts a new goroutine and is not blocking so your output is 5 chunks ahead of the actual download progress. You also have some syntax errors in your code.

I don't really have time right now to work on concat myself because of finals coming up so if you want take another look at it, and I will merge your changes when they work.

Moremackles commented:
I have fixed the previous errors and added a little information to the readme. Thanks for telling me about go, I had never used the language before so apologies for the mistakes.
